### ISSUE TITLE
If HTML formatting is included in titles, it is read out by the screen reader

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#2549]
I think some/all of the instances of `{{compile displayTitle}}` in heading.hbs should be changed to `{{compile title}}` instead.

What I'm not sure is whether it should be necessary to provide a 'fallback' in the case that `title` isn't defined - or assume that it always _should_ be i.e. if it isn't it's an error on the part of the content author.
